pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource_test.go

```diff
@@ -1450,8 +1591,11 @@ func TestBuildingAzureLogAnalyticsQueries(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			queries, err := datasource.buildQueries(ctx, tt.queryModel, dsInfo)
+			queries, err := datasource.buildQueries(ctx, tt.queryModel, dsInfo, tt.fromAlert)
 			tt.Err(t, err)
+			if err != nil {
```

Comment:
I'd rather not return so if any other tests error we see all the errors at once, wdyt?

Reply:
Yes! Fair point, I have removed the logic to return from the function if there are any errors.